src/main/java/pl/allegrotech/weatherapp/infrastructure/repository/RepositoryInit.java

```diff
@@ -0,0 +1,32 @@
+package pl.allegrotech.weatherapp.infrastructure.repository;
+
+import org.springframework.stereotype.Component;
+import pl.allegrotech.weatherapp.domain.Location;
+import pl.allegrotech.weatherapp.domain.Weather;
+import pl.allegrotech.weatherapp.domain.WeatherRepository;
+
+import javax.annotation.PostConstruct;
+
+@Component
+public class RepositoryInit {
+    private final WeatherRepository weatherRepository;
```

Comment:
jakby przenieść logikę z tej klasy do InMemoryWeatherRepository? mogłoby to tak wyglądać:
```
public class InMemoryWeatherRepository implements WeatherRepository {

    private static final List<Weather> initialWeatherList = List.of(
            new Weather(new Location(18, 53), "Toruń", 15),
            new Weather(new Location(23, 53), "Białystok", 9),
            new Weather(new Location(20, 49), "Zakopane", 13),
            new Weather(new Location(14, 53), "Szczecin", 14),
            new Weather(new Location(22, 49), "Wetlina", 11)
    );

    private final Map<Location, Weather> storage = new HashMap<>();

    public InMemoryWeatherRepository() {
        initialWeatherList.forEach(this::save);
    }

    @Override
    public Weather save(Weather weather) {
        storage.put(weather.getLocation(), weather);
        return weather;
    }
    ...
```

Reply:
fajny pomysł, zmienione